src/index.jsx

```diff
@@ -1,16 +1,122 @@
-import { runInContext } from 'vm-shim';
-import { createContext } from 'vm-browserify';
-import Window from 'happy-dom/lib/window/Window';
+import { runInContext } from 'vm-shim'
+import Window from 'happy-dom/lib/window/Window'
 
-import App from './App';
-import Elm from './elm-dist/elm-bundle';
+import App from './App'
+import Button from './Button'
+import Text from './Text'
 
-function init(App) {
+import Elm from './elm-dist/elm-bundle'
+
+const withAttrs = BaseElement =>
```

Comment:
* This `(BaseElement) => class extends BaseElement {}` pattern is a `mixin`.
* These have been a useful way to organize chunks of functionality around a `UIElement`
* I wonder how TypeScript supports mixins 🤔